pkg/selectors/kernel.go

```
@@ -430,6 +439,18 @@ func writeMatchRangesInMap(k *KernelSelectorState, values []string, ty uint32, o
 					protocolStr := fmt.Sprintf("%d", protocol)
 					rangeStr = []string{protocolStr, protocolStr}
 				}
+			case SelectorOpFamily:
+				family, err := network.InetFamilyNumber(v)
+				if err == nil {
+					familyStr := fmt.Sprintf("%d", family)
+					rangeStr = []string{familyStr, familyStr}
+				}
+			case SelectorOpState:
+				state, err := network.TcpStateNumber(v)
+				if err == nil {
+					stateStr := fmt.Sprintf("%d", state)
+					rangeStr = []string{stateStr, stateStr}
+				}
```

ditto for failing silently.

---

Ditto